luthier/config/src/config.lua

```diff
@@ -26,6 +26,10 @@ local dimensionConfig = dimensionUtils.build_dimensions_config(dimensions)
 local metricConfig = metricsUtils.build_metric_config(metrics)
 local tableConfig = tableUtils.build_table_config(tables)
 
+-- make the directory ../external, which can be automated in 
+-- a script since creating a dir in Lua is awkard, in future.
+-- can be circumvented by using the LuaFileSystem module
+os.execute("mkdir -p ../external/")
```

Comment:
This basically locks Fili into being run on a Unixy system, which isn't great.
On the other hand, I *also* don't want to force people to download a bunch of modules
to use the Lua config. With that
mind, I'm wondering if it's better to actually create the empty `external` directory, add
it to git, but ignore its contents. Unfortunately, it looks like doing so requires
a bit of an ugly hack. You need to basically put an empty file (like say `.keep`)
and force git to version control it despite `external` being ignored.

Not ideal, and kinda ugly, but I personally think that's better than imposing Unix
on people running Fili, or requiring people to install luarocks so they can
install a file system module. The other option of course, is that we don't have
an external directory at all. Just dump the JSON files into the `luthier/config`
directory. @mclawhor Thoughts?

Reply:
I would say that this directory should be created in build time as part of the set up process. 
Is there any platform independent way of creating an empty dir as part of the maven instruction?

Reply:
This might give some ideas: https://stackoverflow.com/questions/4085973/create-directory-when-needed-in-maven